perfkitbenchmarker/vm_util.py

```diff
@@ -372,8 +379,18 @@ def IssueCommand(cmd, force_info_log=False, suppress_warning=False, env=None):
   process = subprocess.Popen(cmd, env=env, shell=shell_value,
                              stdout=subprocess.PIPE,
                              stderr=subprocess.PIPE)
+
+  def _KillProcess(process):
+    logging.warning('IssueCommand timed out. Killing Process.')
```

Comment:
Nit: this can take no arguments (`process` is in the enclosing scope).
I think it would also be good to:
* Make this `error` level, rather than `warning`.
* Include the value of `timeout` and `cmd`.